recipes/libvips/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+./configure --prefix="${PREFIX}" \
+	&& make \
```

Comment:
i dont think the && style is needed. conda will run thi script and return on the first faile command

Reply:
> -ftree-vectorize and by default at -O3. To allow vectorization on powerpc* platforms also use -maltivec. On i?86 and x86_64 platforms use -msse/-msse2. To enable vectorization of floating point reductions use -ffast-math or -fassociative-math.

i thinik we already add the sse, but you can add the other flags.

Does llvm work too? or just gcc

Reply:
`-mtune=haswell -ftree-vectorize`
`-march=nocona`

Are already in in the CFLAGS. `nocona implies -msse3`
https://wiki.gentoo.org/wiki/GCC_optimization

You are missing the math flags, which presumably you need?

Reply:
OK, if the vectoriser is on it should be fine. 

I'll remove the `&&`. I guess conda runs this file with `set -e` and then sourcing it, is that right? So I guess I can remove the `#!/bin/bash` too.

Reply:
I usually use `-mnative`, but I assumed conda set something like that already.

Reply:
I forget what mnative does. Unfortunately you don't always control the exact compilation machine, so it is better to explicit about the exact processor you are targeting.

I mean, I'm not sure if the flags will be accepted, but there is a much higher chance of them getting accepted than '-03'

Reply:
I think conda will assume it is bash unless otherwise specified. I mean that shebang line isn't super distracting but the bash one liner isn't too friendly.

Reply:
Dockerfiles have made me very used to huge `x && y && z` lines, it's the standard idiom heh.

I've removed it.